hack/after-build/update-generated-docs.sh

```diff
@@ -39,8 +39,10 @@ shopt -u dotglob
 
 kube::util::gen-analytics "${KUBE_ROOT}"
 
+KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
```

Comment:
Updated

On Wed, Apr 27, 2016 at 7:33 PM, David McMahon <notifications@github.com>
wrote:

> In hack/after-build/update-generated-docs.sh
> <https://github.com/kubernetes/kubernetes/pull/24666#discussion_r61354348>
> :
>
> > @@ -39,8 +39,10 @@ shopt -u dotglob
> >
> >  kube::util::gen-analytics "${KUBE_ROOT}"
> >
> > +KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
>
> Even though it's not necessary used (yet) outside of docs, it's still
> being set and defaulted in >1 place. Definitely prefer to have a single
> include for that in a common place.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24666/files/9f2da6b2b33d3f6fa4047e55b4d835d504612fd5#r61354348>
>